### PR TITLE
INT-8998: refactor compute service

### PR DESCRIPTION
**Pull request by gastonyelmini**

### CODE REVIEW

**Review by RonaldEAM — APPROVED** [review]

File: src/steps/compute/steps/fetch-compute-instances.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+} from '../../../../test/recording';
+import { STEP_COMPUTE_INSTANCES } from '../constants';
+
+describe(`compute#${STEP_COMPUTE_INSTANCES}`, () => {
+  let recording: Recording;
+  afterEach(async () => {
+    if (recording) await recording.stop();
+  });
+
+  test.skip(STEP_COMPUTE_INSTANCES, async () => {
```
Comment (Gonzalo-Avalos-Ribas) [inline src/steps/compute/steps/fetch-compute-instances.test.ts:20]: One more

File: src/steps/compute/steps/fetch-compute-loadbalancers.test.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+} from '../../../../test/recording';
+import { STEP_COMPUTE_LOADBALANCERS } from '../constants';
+
+describe(`compute#${STEP_COMPUTE_LOADBALANCERS}`, () => {
+  let recording: Recording;
+  afterEach(async () => {
+    if (recording) await recording.stop();
+  });
+
+  test.skip(STEP_COMPUTE_LOADBALANCERS, async () => {
```
Comment (Gonzalo-Avalos-Ribas) [inline src/steps/compute/steps/fetch-compute-loadbalancers.test.ts:20]: here aswell

File: src/steps/compute/steps/fetch-compute-region-target-https-proxies.test.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+} from '../../../../test/recording';
+import { STEP_COMPUTE_REGION_TARGET_HTTPS_PROXIES } from '../constants';
+
+describe(`compute#${STEP_COMPUTE_REGION_TARGET_HTTPS_PROXIES}`, () => {
+  let recording: Recording;
+  afterEach(async () => {
+    if (recording) await recording.stop();
+  });
+
+  test.skip(STEP_COMPUTE_REGION_TARGET_HTTPS_PROXIES, async () => {
```
Comment (Gonzalo-Avalos-Ribas) [inline src/steps/compute/steps/fetch-compute-region-target-https-proxies.test.ts:20]: One more .skip

File: src/steps/compute/steps/fetch-compute-ssl-policies.test.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+} from '../../../../test/recording';
+import { STEP_COMPUTE_SSL_POLICIES } from '../constants';
+
+describe(`compute#${STEP_COMPUTE_SSL_POLICIES}`, () => {
+  let recording: Recording;
+  afterEach(async () => {
+    if (recording) await recording.stop();
+  });
+
+  test.skip(STEP_COMPUTE_SSL_POLICIES, async () => {
```
Comment (Gonzalo-Avalos-Ribas) [inline src/steps/compute/steps/fetch-compute-ssl-policies.test.ts:20]: Here!

**Review by Gonzalo-Avalos-Ribas — COMMENTED** [review]
These are all nits

**Pull request merged**